tracing/opentelemetry-tracing-sync/README.md

```diff
@@ -0,0 +1,159 @@
+# OpenTelemetry Sync Tracing for Tapir
+
+Module providing OpenTelemetry integration for Tapir, optimized for synchronous style and virtual threads (Project Loom).
+
+## Installation
+
+```sbt
+libraryDependencies += "com.softwaremill.sttp.tapir" %% "tapir-opentelemetry-tracing-sync" % "version"

```

Comment:
when in docs, you can use `@VERSION@` so that it is replaced at release-time with the correct version